content/1.4-migration.md

```diff
@@ -64,3 +64,44 @@ if (Meteor.isClient) {
 ```
 
 One place this is particularly useful is in [test files that are only intended to run on the client or the server](https://github.com/meteor/todos/commit/3963a65d96cd7ef235a95d5e3a331d6f0606f70f) &mdash; you can now use `import` wherever you like, without having to organize your tests in `client` or `server` directories.
+
+<h2 id="gotchas">Gotchas!</h2>
+Here is a list of observed gotchas...
+
+<h3 id="mongo-3_2-gotchas">Mongo 3.2</h3>
+
+If you are using [oplog tailing](https://github.com/meteor/docs/blob/version-NEXT/long-form/oplog-observe-driver.md) and you see a failed authentication you may need to upgrade to [SCRAM-SHA-1](https://docs.mongodb.com/manual/release-notes/3.0-scram/#upgrade-mongodb-cr-to-scram), essentially: `use admin, db.adminCommand({authSchemaUpgrade: 1});`.  You may need to delete and re-add your oplog reader user.
+
+<h3 id="node-gyp-gotchas">node-gyp</h3>
+
+Make sure this is installed so you can compile your node modules to native if necessary.  See [Install `node-gyp`](https://github.com/nodejs/node-gyp#installation)
+
+<h3 id="clear-out-oldstuff-gotchas">Clear out old stuff</h3>
+
+Try a `meteor reset` or better yet `rm -rf .meteor/local`.  Follow this with the [rebuild command](https://guide.meteor.com/1.4-migration.html#binary-packages-require-build-toolchain).
+
+<h3 id="npm-bcrypt-gotchas">npm-bcrypt</h3>
+
+You may run into this [`npm-bcrypt`](http://stackoverflow.com/questions/29340510/bcrypt-is-breaking-my-meteor-application-how-do-i-fix-it/29347616) problem
+
+<h3 id="debugger-gotcha">debugger statement issue</h3>
+If you have `debugger` statements in your code they will now hit the breakpoint even without a debugger attached.
```

Comment:
agree, but the subtle change to the way the debugger statements worked is fundamental to my debug strategies.  I like the new way it works and I think it will save time.  I am ok with the more detailed info in the forum.